bramkouwen/Rock-Paper-Scissors-Game
# Feature code cleanup

## What does this PR do?
* It cleans up the code in game.py
* It added more files to the project 

## How can I check this PR?
* Pull branch down
* Check moved code

## Screenshots 
None.
